specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json

```diff
@@ -32,7 +32,7 @@
     "application/json;odata=nometadata"
   ],
   "produces": [
-    "application/json;odata=nometadata"
+    "application/json;odata=minimalmetadata"
```

Comment:
I believe @lmazuel suggested in an email a while back we make this nometadata by default since metadata is odata only and not information on the resource. Did the thinking change here?

Reply:
In this case the metadata includes information on how the data should be deserialized along with Etag information on the entity, so I would prefer to include it by default.
Customers can then opt-out if they need high-perf.

Reply:
ok thanks for confirming